live-examples/css-examples/basic-box-model/width.css

```diff
@@ -0,0 +1,6 @@
+#example-element {
+    background-color: #E4F0F5;
+    height: 80%;
+    justify-content: center;
+    color: #000000;
+}
```

Comment:
It would be nice to vertically-center the text. You could borrow what the [border-radius example](https://github.com/mdn/interactive-examples/blob/master/live-examples/css-examples/backgrounds-and-borders/border-radius.css) does, by adding:

```
 display: flex;
 justify-content: center;
 flex-direction: column;
```

You *might* like to use the same color scheme that border-radius uses, too. But that's entirely up to you.

Reply:
Yes, the colour scheme on border-radius _is_ better.